app/forms/nsm/steps/disbursement_cost_form.rb

```diff
@@ -4,13 +4,13 @@ class DisbursementCostForm < ::Steps::BaseFormObject
       include DisbursementCosts
       attr_writer :apply_vat
 
-      attribute :miles, :decimal, precision: 10, scale: 3
-      attribute :total_cost_without_vat, :decimal, precision: 10, scale: 2
+      attribute :miles, :fully_validatable_decimal, precision: 10, scale: 3
+      attribute :total_cost_without_vat, :fully_validatable_decimal, precision: 10, scale: 2
```

Comment:
You could use `:gbp` here, which has the advantage of being forgiving of if providers enter the "£" symbol in the field itself